definitions/features/pulpcore.rb

```diff
@@ -11,10 +11,8 @@ class Features::Pulpcore < ForemanMaintain::Feature
   end
 
   def services
-    self.class.pulpcore_common_services + configured_workers + [
-      system_service('rh-redis5-redis', 5),
-      system_service('httpd', 30)
-    ]
+    self.class.pulpcore_common_services + configured_workers +
+      feature(:redis).services + feature(:apache).services
```

Comment:
Makes me wonder if features should have some kind of "dependent" features (in this case pulpcore depends on redis and apache), thus automatically inheriting the services etc?

Reply:
I heard you yesterday saying "it's too complicated", and that sounds fair :)

Reply:
Yes, I can see there is for_feature but its not something we can use for this purpose. Maybe this is something for future